modules/ocf/manifests/ssl/bundle.pp

```diff
@@ -37,10 +42,17 @@
         target => $cert_path,
         mode   => '0644';
 
+      "/etc/ssl/private/${title}.fullchain":
```

Comment:
The bundle at `/etc/ssl/private/${title}.bundle` should have the same contents as this:

```
jvperrin@lethe:~$ sudo diff /etc/ssl/private/lethe.ocf.berkeley.edu.bundle /etc/ssl/private/lethe.ocf.berkeley.edu.fullchain
43a44
>
```

(except some whitespace changes apparently)

Maybe we should not be using concat and could just link to the fullchain cert instead? Seems cleaner and I think we can remove incommon support now anyway.

Actually, wouldn't using the bundle file fix your permissions issues anyway? I do like that the owner/group can be passed in now, but I don't think we'd need to use it here given the bundle by default is owned by `root:ssl-cert`

Reply:
while it works for the certs itself, unfortunately, the key file has the same restrictive rules, so managing the files themselves still seems necessary :(